docs/developers-guide/snowplow.md

```diff
@@ -0,0 +1,47 @@
+# Snowplow integration
+
+Metabase uses [Snowplow](https://snowplowanalytics.com/) and [SnowcatCloud](https://www.snowcatcloud.com/) for
+collecting analytics data. We only send events when `Anonymous tracking` is enabled. We do not record personal
+identifiable information with the events.
+
+## How to test events manually
+
+We use [Snowplow Micro](https://github.com/snowplow-incubator/snowplow-micro) for end-to-end tests.
+
+1. Run `docker compose up` inside `/snowplow` folder to spin up a local instance.
+2. Start Metabase with `MB_SNOWPLOW_AVAILABLE=true` and anonymous tracking enabled.
+3. Recorded events are available at `http://localhost:9090/micro/`.
```

Comment:
I don't think `/micro/` alone will show anything. `/micro/all` will show the good/bad event counts and `/micro/good` and `/micro/bad` will show the individual events.

Reply:
Also, this was on port 9095 before, right? I'll have to change the default development url to 9090 in the backend PR

Reply:
Yeah, `9090` is exposed by the snowplow-micro docker image and it makes sense to me switch to default. I don't remember why I used `9095` before tbh.